phpstan.neon

```diff
@@ -0,0 +1,8 @@
+parameters:
+    level: 6
+    paths:
+        - src
+    phpVersion: 70200
```

Comment:
```suggestion
    phpVersion: 80300
```

If you want to maintain your sanity, don't fix static analysis for PHP from the last decade.